src/main/kotlin/io/emeraldpay/dshackle/upstream/DefaultNearMethods.kt

```diff
@@ -0,0 +1,63 @@
+package io.emeraldpay.dshackle.upstream
+
+import io.emeraldpay.dshackle.quorum.AlwaysQuorum
+import io.emeraldpay.dshackle.quorum.BroadcastQuorum
+import io.emeraldpay.dshackle.quorum.CallQuorum
+import io.emeraldpay.dshackle.upstream.calls.CallMethods
+import io.emeraldpay.dshackle.upstream.ethereum.rpc.RpcException
+
+class DefaultNearMethods : CallMethods {
+
+    private val all = setOf(
+        "view_access_key",
+        "query",
+        "EXPERIMENTAL_changes",
+        "block",
+        "chunk",
+        "EXPERIMENTAL_changes_in_block",
+        "gas_price",
+        "status",
+        "network_info",
+        "validators",
+        "tx",
+        "EXPERIMENTAL_tx_status",
+        "EXPERIMENTAL_receipt",
+    )
+
+    private val add = setOf(
+        "broadcast_tx_async",
+        "broadcast_tx_commit",
+    )
+
+    private val allowedMethods: Set<String> = all + add
+
+    override fun createQuorumFor(method: String): CallQuorum {
+        return when {
+            add.contains(method) -> BroadcastQuorum()
+            all.contains(method) -> AlwaysQuorum()
```

Comment:
It looks like we can omit this line if we have else branch

Reply:
will fix